### DISCUSSION TITLE
How to start a Docker Container

### DISCUSSION THREAD

**Discussion by 7aklhz** [caprover/caprover#1379] [Q&A] [answered]
Hi,

Up until now, I've used Docker Hub to pull a docker image and then env variables to make things running. 

Currently, I want to make Selenium running. I've pulled their image (selenium/hub:latest) but there are not env variables but I need to make some changes/add specifications. On their page I have the following information 
Start a docker container : 
docker run -d -p 4444:4444 -p 7900:7900 --shm-size="2g" selenium/standalone-firefox:4.1.2-20220317

How is this doable/"translated" in caprover ? Where should I input "7900", "shm-size='2g'" and "selenium/standalone-firefox:4.1.2-20220317" ? 

Thanks

**Comment by githubsaturn** [comment, marked as answer]
Multiple methods:
- You can simply just run the command that they provided. CapRover doesn't touch the containers that are launched via other methods.
- If you want to deploy it under CapRover, you'll have to use port mapping for `p 4444:4444 -p 7900:7900` . But there is no native support for `shm-size` in CapRover.
